### ISSUE TITLE
Create a pc2ccs.github.io repo and populate it via pc2v9 .gitlab-ci.yaml

### ISSUE THREAD

**Issue by troy2914** [pc2ccs/pc2v9#65]
This is a follow on to the builds and nightly-builds ticket #50.
The intent is to a create a site like icpctools/cpcptools.github.io

A new target will be added to .gitlab-ci.yaml, following the icpctools/icpctools .gitlab-ci.yaml file.

The intent of this pc2ccs.github,.io is to be linked from icpc.global

**Comment by troy2914** [comment]
This is failing because the image we are using does not have sudo.

**Comment by troy2914** [comment]
Now it's is failing on a 

```
$ website/scripts/populate-releases.py
  File "website/scripts/populate-releases.py", line 76
    print(f"ERROR: {asset['name']} failed to match {assetPattern}\n")
                                                                   ^
SyntaxError: invalid syntax
```

guessing this image has an old python3.

**Comment by troy2914** [comment]
sed -i "" failed on this image, but is required when running on macos.

**Comment by troy2914** [comment]
Pulling docker image pc2fromecs:jdk8hugo ...
[0;msection_end:1589349069:prepare_executor
[0K[31;1mERROR: Job failed: Error response from daemon: pull access denied for pc2fromecs, repository does not exist or may require 'docker login' (docker.go:198:0s)
[0;m

fixed in PR#88
